apps/core-v2-testnet/tsconfig.json

```diff
@@ -6,8 +6,12 @@
     "outDir": "dist",
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": ".",
+    "paths": {
+      "@utils/*": ["utils/*"]
```

Comment:
Do we need this? Seems like we should just use `rootDir` and avoid the path aliases here.

Reply:
yeah we don't need this just removed it!